### ISSUE TITLE
Proposal: Switch to `cross-fetch` instead of `isomorphic-fetch`

### ISSUE THREAD

**Issue by jacekradko** [mockyeah/mockyeah#635]
The `isomorphic-fetch` library has not been maintained for years and it seems to be abandoned.  The spiritual successor appears to be `cross-fetch` and it is currently being maintained. It would be great to switch. At the moment if you don't have `isomorphic-fetch` as a dependency in your project, `mockyeah` throws the following error:

```
ReferenceError: Response is not defined
	at node_modules/@mockyeah/fetch/dist/main.js:1:8797
```

**Comment by AndersDJohnson** [comment]
@jacekradko Good idea, thanks!

Are you seeing this error using `@mockyeah/fetch` alone? I think that's expected because using it alone would require your project only run in an environment with `fetch` and the associated classes like `Response` available globally, or have installed a global polyfill. I think `cross-fetch` can polyfill globally too but you must `import 'cross-fetch/polyfill'`.

If instead you're seeing it with `@mockyeah/server` (which depends on `@mockyeah/fetch`), then that's not expected, because `@mockyeah/server` already includes `isomorphic-fetch` which installs those classes as a global. But either way, I'll see if we can switch `@mockyeah/server` to `cross-fetch` global polyfill since that's maintained.

**Comment by AndersDJohnson** [comment]
@jacekradko Okay, I've published `@mockyeah/server@1.0.3` with `cross-fetch` instead of `isomorphic-fetch`. Let me know if it doesn't work for you. Thanks!
